python/doc/theory/data_analysis/bic.rst

```diff
@@ -3,16 +3,16 @@
 Bayesian Information Criterion (BIC)
 ------------------------------------
 
-This method deals with the modelling of a probability distribution of a
+This method defines the probability distribution of a
```

Comment:
I don't understand the change, it was better before IMO. How does the BIC "define" the probability distribution of a random vector?

Reply:
OK, I will fix this.

Reply:
Fixed at ca862954b